### ISSUE TITLE
num_threads or nThreads?

### ISSUE THREAD

**Issue by whartley94** [richzhang/colorization-pytorch#13]
I've noticed that the base options for this repository uses opt.nThreads, but I can't actually see anywhere this is used.

I've also done some digging and in the data folder init.py the CustomDatasetLoader calls opt.num_threads, which I believe is what the pix2pix repo uses in their base options, although I don't think this is used either in this project? 

Is the data loading for this project actually multi-threaded?
In train.py torch.utils.data.DataLoader(dataset, batch_size=opt.batch_size, shuffle=True) is called, should I change this to include num_workers=int(opt.nThreads) if I want to speed up the data loading?

Thanks for the amazing code/project! :)

**Comment by junyanz** [comment]
It is `num_threads`. See this [line](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/options/base_options.py#L44)

**Comment by whartley94** [comment]
Ah, so I was referring to this [line](https://github.com/richzhang/colorization-pytorch/blob/master/options/base_options.py#30), has it been ported over correctly? Hope I'm posting on the right repo?

**Comment by junyanz** [comment]
Sorry that I thought it was a question for pix2pix/cyclegan repo. Yes, changing all the flags to `num_threads` should address the issue. I will update the code.

**Comment by whartley94** [comment]
No worries, I'm doing a local version at the moment, assuming i'll need to add it to the args [here?](https://github.com/richzhang/colorization-pytorch/blob/master/train.py#L32)

**Comment by junyanz** [comment]
I updated the code.

**Comment by whartley94** [comment]
Thanks for sorting that, could you help me out, I'm still a bit confused whether passing num_threads will actually change anything if I call train.py --num_threads = 2 (for example).

My current understanding is that this would need to be passed [here](https://github.com/richzhang/colorization-pytorch/blob/master/train.py#L32) as num_workers? Am I missing something? 

Thanks :)

**Comment by junyanz** [comment]
Yes, you are right. I will update the code accordingly.
